xml/System.Windows.Forms/MessageBoxIcon.xml

```diff
@@ -82,7 +82,22 @@ Private Sub Form1_FormClosing( _
     End If  
 End Sub  
 ```  
-  
+
+Most of the icons are the same as at least one other. They also look different
+on different versions of Windows. Here's what each looks like on Windows 10:
+
+![messagebox-asterisk](messagebox-asterisk.png)  
```

Comment:
@nic-hartley Hi! Thank you for this pull request. 

These **.gif** files you replaced are used by another article in the WPF documentation. They are actually doing what you're trying to do 😄  You can see that page over [here](https://docs.microsoft.com/en-us/dotnet/api/system.windows.messageboximage?view=netframework-4.7.2).

Since I have the doc source for that page open, this is what they are doing:

```markdown
|||
|-|-|
|![Hand icon](~/add/media/messagebox-hand.gif "Hand icon")|Hand|
|![Question icon](~/add/media/messagebox-question.gif "Question icon")|Question|
|![Exclamation icon](~/add/media/messagebox-exclamation.gif "Exclamation icon")|Exclamation|
|![Asterisk icon](~/add/media/messagebox-asterisk.gif "Asterisk icon")|Asterisk|
|![Stop icon](~/add/media/messagebox-stop.gif "Stop icon")|Stop|
|![Error icon](~/add/media/messagebox-error.gif "Error icon")|Error|
|![Warning icon](~/add/media/messagebox-warning.gif "Warning icon")|Warning|
|![Information icon](~/add/media/messagebox-information.gif "Information icon")|Information|
```

Having screenshots with text on it is not very good for localization. All that text in the screenshots would have to be translated into other languages. It would be best if you could update the original .gif files with the newer icons and then add this icon table here in this article.